flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/BDREHDFSEventSink.java

```diff
@@ -356,6 +356,7 @@ static CompressionCodec getCodec(String codecName) {
    * HDFS. <br/>
    * This method is not thread safe.
    */
+  @Override
   public Status process() throws EventDeliveryException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "process" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)